src/views/InvolvementProfile/components/Membership/components/MemberList/components/MemberListItem/index.js

```diff
@@ -231,7 +232,7 @@ const MemberListItem = ({
           cancelButtonClicked={() => setIsRemoveAlertOpen(false)}
         >
           Are you sure you want to remove {member.FirstName} {member.LastName} (
-          {member.ParticipationDescription}) from this involvement?
+          {member.ParticipationDescription.trim()}) from this involvement?
```

Comment:
Should we be trimming this on the backend?

Reply:
I don't know. I just noticed a problem and fixed it.